src/include/sof/lib/cpu-clk-manager.h

```diff
@@ -0,0 +1,57 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Krzysztof Frydryk <frydryk.krzysztof@intel.com>
+ */
+
+/**
+  * \file include/sof/lib/cpu-clk-manager.h
+  * \brief CPU clk manager definition
+  * \author Krzysztof Frydryk <krzysztofx.frydryk@intel.com>
+  */
+
+#ifndef __SOF_LIB_CPS_BUDGET_MANAGER_H__
```

Comment:
Pls update the pre-proc symbols to reflect the new (nice) name.

Reply:
Sorry for the late answer, sure. Missed that.